### PR TITLE
[Xcodeproj] Added Package.swift to the root of generated Xcode projects

### PR DESCRIPTION
**Pull request by czechboy0**
This should make it easier to edit `Package.swift` from the generated Xcode project.

screenshot 2016-03-19 19 30 40

### CODE REVIEW

File: Sources/Xcodeproj/pbxproj().swift [hunk 1]
```diff
@@ -143,7 +153,7 @@ public func pbxproj(srcroot srcroot: String, projectRoot: String, modules: [Swif
         print("            target = \(module.targetReference);")
         print("        };")
     }
-
+
```
Comment (kostiakoval) [inline Sources/Xcodeproj/pbxproj().swift]: remove white-space

Comment (czechboy0) [inline Sources/Xcodeproj/pbxproj().swift]: Done :+1:

**Comment by kostiakoval** [comment]
LGTM

**Comment by mxcl** [comment]
@swift-ci Please test and merge

**Comment by kostiakoval** [comment]
@mxcl  Can CI merge PR on it's own ?

**Comment by mxcl** [comment]
@swift-ci Please test

**Comment by mxcl** [comment]
I’m surprisingly excited for all these Xcode project additions.

File: Sources/Xcodeproj/Module+PBXProj.swift [hunk 2]
```diff
@@ -70,6 +70,11 @@ private func fileRef(suffixForModuleSourceFile path: String, srcroot: String) ->
     }.joined(separator: "")
 }
 
+func fileRef(inProjectRoot name: String, srcroot: String) -> (String, String, String) {
```
Comment (aciidb0mb3r) [inline Sources/Xcodeproj/Module+PBXProj.swift]: named tuple seems better to me for the return type

File: Sources/Xcodeproj/pbxproj().swift [hunk 3]
```diff
@@ -153,7 +153,7 @@ public func pbxproj(srcroot srcroot: String, projectRoot: String, modules: [Swif
         print("            target = \(module.targetReference);")
         print("        };")
     }
-    
+
```
Comment (aciidb0mb3r) [inline Sources/Xcodeproj/pbxproj().swift]: should probably squash this

**Pull request merged**